airflow/api_fastapi/views/public/dags.py

```diff
@@ -92,6 +93,28 @@ async def get_dags(
     )
 
 
+@dags_router.get(
+    "/tags",
+    response_model=list[DAGTagResponse],
```

Comment:
Is response_model required when we are also including return type annotation? If so, I can also add to the PRs I am working on. So, for already merged PRs, I don't see response_model.

Reply:
I think this is not needed. ResponseModels are already automatically generated in FastAPI and cascaded to openapi spec as well. 
I think rather than adding this here, we should create a pydantic model for this response and retrun according to the new model.
Similar to this [DAGCollectionResponse](https://github.com/apache/airflow/blob/b92c66d45d206c670516f192662bfcf8ad34bec8/airflow/api_fastapi/serializers/dags.py#L103) model but without `total_entries` of course